storage/stow_store_test.go

```diff
@@ -143,6 +149,81 @@ func TestAwsBucketIsNotFound(t *testing.T) {
 	})
 }
 
+func TestStowStore_CreateSignedURL(t *testing.T) {
+	labeled.SetMetricKeys(contextutils.ProjectKey, contextutils.DomainKey, contextutils.WorkflowIDKey, contextutils.TaskIDKey)
+
+	const container = "container"
+	t.Run("Happy Path", func(t *testing.T) {
+		testScope := promutils.NewTestScope()
+		fn := fQNFn["s3"]
+		s, err := NewStowRawStore(fn(container), &mockStowLoc{
+			ContainerCb: func(id string) (stow.Container, error) {
+				if id == container {
+					return newMockStowContainer(container), nil
+				}
+				return nil, fmt.Errorf("container is not supported")
+			},
+			CreateContainerCb: func(name string) (stow.Container, error) {
+				if name == container {
+					return newMockStowContainer(container), nil
+				}
+				return nil, fmt.Errorf("container is not supported")
+			},
+		}, false, testScope)
+		assert.NoError(t, err)
+
+		actual, err := s.CreateSignedURL(context.TODO(), DataReference("https://container/path"), SignedURLProperties{})
```

Comment:
i'm confused why this happy path starts with https://... shouldn't this be s3 or gcs?

Reply:
Any valid url format is ok.. no?

Reply:
i guess, i just find it confusing.  would this ever be called like this?